app/controllers/users_controller.rb

```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
```

Comment:
- [ ] Your application is not properly routed. Kindly add a  route to the _root_ of your project. Also, kindly ensure to properly configure routes project route show posts, or make it easily navigable.

**Improperly configured routes:**
![image](https://user-images.githubusercontent.com/12745474/146591527-b49c0d08-a18b-436a-a545-9e3cef7bbc2c.png)

Reply:
Fixed